README.md

```diff
@@ -1302,8 +1302,9 @@ bucket if it doesn't already exist and only write files to the specified path.
 
 ### Built-in Functions
 
-Terragrunt allows you to use built-in functions anywhere in `terragrunt.hcl`, just like Terraform! The functions 
-currently available are: 
+Terragrunt allows you to use built-in functions anywhere in `terragrunt.hcl`, just like Terraform!  All of the
+[built-in functions from Terraform](https://www.terraform.io/docs/configuration/functions.html) are available, as well
+as the following Terragrunt-specific functions:
```

Comment:
To make this even more visible, consider adding a bullet point right at the top called "Terraform built-in functions", and add a corresponding subsection for it that links to those docs and shows an example of using a couple of those functions.

Reply:
Will do.

Reply:
Done.